woocommerce/woocommerce-ios
# Remove placeholder content for store stats / top performers sections for each time range tab ASAP

Part of #5742 


### Description


Previously, when syncing store stats (order/visitor stats) and top performers, we show the loading state for the whole dashboard screen until all the API calls are complete. With Home Screen M2, we are showing the latest data as soon as they come back. This means that we are removing the placeholder content whenever the API sync call for the section and time range tab is complete. The app could "feel" less slow with this change, and the time range tab is interactive all the time now 🙂 

Please note that the placeholder for the "Top Performers" label will be implemented separately.

### Testing instructions


I find that UI gets stuck when launching the app in simulators but not on devices (I paused the app but didn't find anything obvious). Feel free to test it on a device if it's easier.

- Launch the app, and quickly tap other time range tabs (This Week / This Month / This Year) --> the placeholder content should be shown first, then the store stats and top performers sections become visible ASAP
- Switch to another store --> the placeholder content should be shown for the new store
- Go to any other tab in the bottom tab bar, then go back to the My Store tab --> the placeholder content should look the same (without calling `placeholderChartsView.restartGhostAnimation(style: .wooDefaultGhostStyle)` in `viewWillAppear`, the placeholder content stops animating after leaving the screen and coming back)

### Screenshots



https://user-images.githubusercontent.com/1945542/148337555-4b2da44e-7750-4f7e-ae3e-25f5a28dc300.mov



---
